src/content/content.component.ts

```diff
@@ -17,97 +25,57 @@ import { BUTTON_TYPE } from '@shared-components/button/button.component';
 export class ContentComponent implements OnInit {
   value = '';
   SORT = SORT;
-  sortType: SORT = SORT.ASC;
   DISPLAY_TYPE = DISPLAY_TYPE;
   BUTTON_TYPE = BUTTON_TYPE;
   type: DISPLAY_TYPE = DISPLAY_TYPE.LIST;
   onlyFavoriteMovie = false;
+  allMovies: Movie[] = [];
   page: PageEvent = {
     pageIndex: DEFAULT_PAGE_INDEX,
     pageSize: DEFAULT_PAGE_SIZE,
     length: this.allMovies.length,
   };
   managedMovies: Movie[] = [];
 
-  constructor(public apiService: ApiService,
-              public moviesService: MoviesService,
-              public snackBar: MatSnackBar) {
-              }
-
+  constructor(public snackBar: MatSnackBar,
+              private readonly store: Store<AppState>) {
+  }
 
   ngOnInit(): void {
-    this.moviesService.setMoviesFromLocalStorage();
-    this.managedMovies = this.allMovies;
+    this.store.dispatch(new DownloadDataFromLocalStorage());
+    this.store.select(getAllMovies).subscribe(allMovies => {
```

Comment:
warto zadbać o unsubscribe żeby nie tworzyły się memory leaks: 
https://medium.com/@sub.metu/angular-how-to-avoid-memory-leaks-from-subscriptions-8fa925003aa9

możesz do tego użyć tej libki - https://github.com/ngneat/until-destroy